dags/bqetl_kpi_dashboard.py

```diff
@@ -54,6 +54,64 @@
         dag=dag,
     )
 
+    wait_for_smoot_usage_desktop_v2 = ExternalTaskSensor(
+        task_id="wait_for_smoot_usage_desktop_v2",
+        external_dag_id="main_summary",
+        external_task_id="smoot_usage_desktop_v2",
+    )
+
+    telemetry_derived__smoot_usage_new_profiles__v2.set_upstream(
+        wait_for_smoot_usage_desktop_v2
+    )
+
+    wait_for_smoot_usage_fxa_v2 = ExternalTaskSensor(
+        task_id="wait_for_smoot_usage_fxa_v2",
+        external_dag_id="fxa_events",
+        external_task_id="smoot_usage_fxa_v2",
+    )
+
+    telemetry_derived__smoot_usage_new_profiles__v2.set_upstream(
+        wait_for_smoot_usage_fxa_v2
+    )
+
+    wait_for_smoot_usage_nondesktop_v2 = ExternalTaskSensor(
+        task_id="wait_for_smoot_usage_nondesktop_v2",
+        external_dag_id="copy_deduplicate",
+        external_task_id="smoot_usage_nondesktop_v2",
+    )
+
+    telemetry_derived__smoot_usage_new_profiles__v2.set_upstream(
+        wait_for_smoot_usage_nondesktop_v2
+    )
+
     telemetry_derived__smoot_usage_new_profiles_compressed__v2.set_upstream(
         telemetry_derived__smoot_usage_new_profiles__v2
     )
+
+    wait_for_firefox_accounts_exact_mau28_raw = ExternalTaskSensor(
+        task_id="wait_for_firefox_accounts_exact_mau28_raw",
+        external_dag_id="fxa_events",
+        external_task_id="firefox_accounts_exact_mau28_raw",
+    )
+
+    telemetry__firefox_kpi_dashboard__v1.set_upstream(
+        wait_for_firefox_accounts_exact_mau28_raw
+    )
+
+    wait_for_exact_mau_by_dimensions = ExternalTaskSensor(
+        task_id="wait_for_exact_mau_by_dimensions",
+        external_dag_id="main_summary",
+        external_task_id="exact_mau_by_dimensions",
+    )
+
+    telemetry__firefox_kpi_dashboard__v1.set_upstream(wait_for_exact_mau_by_dimensions)
+
+    wait_for_firefox_nondesktop_exact_mau28 = ExternalTaskSensor(
+        task_id="wait_for_firefox_nondesktop_exact_mau28",
+        external_dag_id="copy_deduplicate",
+        external_task_id="firefox_nondesktop_exact_mau28",
+    )
+
+    telemetry__firefox_kpi_dashboard__v1.set_upstream(
+        wait_for_firefox_nondesktop_exact_mau28
+    )
```

Comment:
Looking at https://github.com/mozilla/telemetry-airflow/blob/master/dags/kpi_dashboard.py, there are a lot more dependencies here which might not be strictly necessary but shouldn't do any harm either?